### ISSUE TITLE
Replace deprecated "*.symbols.nupkg" files with "*.snupkg"

### ISSUE THREAD

**Issue by jakhog** [dolittle-tools/DotNET.Common#4]
Other tooling - e.g. the _NuGet CLI_ - works a lot better with this new file extension format. It also makes globbing easier in shell scripts, so I see no reason why we should not enable this option.

We just need to add `<SymbolPackageFormat>snupkg</SymbolPackageFormat>` in the `Dolittle.Common.props` file to enable this by default.

**Comment by einari** [comment]
We then need to update our release from local script sitting in the build repository, as it's actually publishing the *.symbols.nuget file in an attempt to get debugging to work.

**Comment by jakhog** [comment]
Yes, let me try it locally first. But it seems that the _NuGet CLI_ works when publishing both packages - if you use the `*.snupkg` extension.

**Comment by einari** [comment]
*NuGet CLI* will probably do the right thing. Its just our scripts that are wrong - we have been explicitly publishing the `*.symbols.nupkg`. So right now the [release_from_local.sh](https://github.com/dolittle-tools/DotNET.Build/blob/master/release_from_local.sh) script has an explicit **first all symbol packages** then **all regular packages**.

**Comment by woksin** [comment]
Wouldn’t that local script be massively improved by using nuget init and nuget push to push the packages to the local folder instead

**Comment by jakhog** [comment]
Yep, it probably could be simplified significantly with our learnings fromt the build pipelines. Hopefully that could make it more multi OS friendly as well.

**Comment by woksin** [comment]
Someone should make an issue for that :)
